src/v2/Apps/Order/Mutations/shippingMutations.ts

```diff
@@ -0,0 +1,149 @@
+import { graphql } from "react-relay"
+import { AddressModalMutation } from "v2/__generated__/AddressModalMutation.graphql"
+import {
+  UserAddressAttributes,
+  ShippingCreateUserAddressMutation,
+} from "v2/__generated__/ShippingCreateUserAddressMutation.graphql"
+import { ShippingOrderAddressUpdateMutation } from "v2/__generated__/ShippingOrderAddressUpdateMutation.graphql"
+import { CommitMutation } from "../Utils/commitMutation"
+import {
+  convertShippingAddressToMutationInput,
+  convertShippingAddressForExchange,
+} from "../Utils/shippingAddressUtils"
+
+export const setShippingMutation = (
+  commitMutation: CommitMutation,
+  variables: ShippingOrderAddressUpdateMutation["variables"]
+) => {
+  return commitMutation<ShippingOrderAddressUpdateMutation>({
+    variables,
+    // TODO: Inputs to the mutation might have changed case of the keys!
+    mutation: graphql`
+      mutation ShippingOrderAddressUpdateMutation(
+        $input: CommerceSetShippingInput!
+      ) {
+        commerceSetShipping(input: $input) {
+          orderOrError {
+            ... on CommerceOrderWithMutationSuccess {
+              __typename
+              order {
+                internalID
+                state
+                requestedFulfillment {
+                  __typename
+                  ... on CommerceShip {
+                    name
+                    addressLine1
+                    addressLine2
+                    city
+                    region
+                    country
+                    postalCode
+                    phoneNumber
+                  }
+                }
+              }
+            }
+            ... on CommerceOrderWithMutationFailure {
+              error {
+                type
+                code
+                data
+              }
+            }
+          }
+        }
+      }
+    `,
+  })
+}
+
+export const saveUserAddressMutation = (
```

Comment:
(nonblocking thought) Can each of these mutations go in their own file in the `Mutations` directory so that in the future, at a glance, devs can see what changes just by looking at the file tree?